### PR TITLE
[SW-1988] Deprecate passing arguments via kwargs method in getOrCreate in PySparkling

### PR DESCRIPTION
**Pull request by jakubhava**
See https://0xdata.atlassian.net/browse/SW-1988 for more details

### CODE REVIEW

File: doc/src/site/sphinx/tutorials/secured_flow.rst [hunk 1]
```diff
@@ -68,18 +68,48 @@ In order to use https correctly, the following two options need to be specified:
...
+
+            library(rsparkling)
+            sc <- spark_connect(master = "local")
+            conf <- H2OConf()$setJks("/path/to/keystore")$setJksPass("password")
+            hc <- H2OContext.getOrCreate(conf)
+
+
+In case your certificates are self-signed, the connection to the H2O cluster will fail due to the security
+limitations. In this case, you can skip the certificates verification
+by calling ``setVerifySslCertificates`` ``H2OConf`` as:
```
Comment (mn-mikke) [inline doc/src/site/sphinx/tutorials/secured_flow.rst]: by calling ``setVerifySslCertificates`` on ``H2OConf``?

File: doc/src/site/sphinx/tutorials/secured_flow.rst [hunk 2]
```diff
@@ -113,31 +143,54 @@ certificates are created.
...
+        :title: R
+
+        To enable https in RSparkling using this mode, run in your RStudio:
+
+
+        and when you have the shell running, start ``H2OContext`` as:
+
+        .. code:: r
+
+            from pysparkling import *
```
Comment (mn-mikke) [inline doc/src/site/sphinx/tutorials/secured_flow.rst]: `pysparkling` seems to be a wrong namespace for R.

File: doc/src/site/sphinx/tutorials/secured_flow.rst [hunk 3]
```diff
@@ -113,31 +143,54 @@ certificates are created.
...
+            hc = H2OContext.getOrCreate()
+
+        You can also start PySparkling shell without the configuration
```
Comment (mn-mikke) [inline doc/src/site/sphinx/tutorials/secured_flow.rst]: `PySparkling` -> `RSparkling`?

File: doc/src/site/sphinx/tutorials/secured_flow.rst (hunk 2 above)
Comment (jakubhava) [reply doc/src/site/sphinx/tutorials/secured_flow.rst]: Yup, thanks, rebase error

File: r/src/R/ai/h2o/sparkling/H2OContext.R [hunk 4]
```diff
@@ -61,10 +61,9 @@ H2OContext.getOrCreate <- function(sc = NULL, conf = NULL) {
   port <- invoke(jhc, "h2oLocalClientPort")
   if (!isClientConnected(jhc)) {
     if (contextPath == "") {
-      invisible(capture.output(h2o.init(strict_version_check = FALSE, ip = ip, port = port, startH2O = F, username = conf$userName(), password = conf$password())))
-    } else {
-      invisible(capture.output(h2o.init(strict_version_check = FALSE, ip = ip, port = port, context_path = contextPath, startH2O = F, username = conf$userName(), password = conf$password())))
+        contextPath <- NA_character_
     }
+    invisible(capture.output(h2o.init(strict_version_check = FALSE, ip = ip, port = port, context_path = contextPath, startH2O = F, username = conf$userName(), password = conf$password())))
```
Comment (jakubhava) [inline r/src/R/ai/h2o/sparkling/H2OContext.R]: TODO: currently RSparkling does not handle the verify_ssl_certificates, pass it to the h2o.init()

**Pull request merged**